src/etl/NFTHelpers.cpp

```diff
@@ -47,6 +47,16 @@
 
 namespace etl {
 
+std::pair<std::vector<NFTTransactionsData>, std::optional<NFTsData>>
+getNFTokenMofidyData(ripple::TxMeta const& txMeta, ripple::STTx const& sttx)
+{
+    auto tx = NFTTransactionsData(sttx.getFieldH256(ripple::sfNFTokenID), txMeta, sttx.getTransactionID());
+
+    auto const tokenID = sttx.getFieldH256(ripple::sfNFTokenID);
+    // note: sfURI is optional, if it is absent, we will update the uri as empty string
+    return {{tx}, NFTsData(tokenID, txMeta, sttx.getFieldVL(ripple::sfURI))};
```

Comment:
We probably want to `std::move` the `tx` here or even avoid the temporary at all, wdyt?